std/assembly/index.d.ts

```diff
@@ -1165,12 +1164,12 @@ interface ArrayBufferView<T> {
 }
 
 /* @internal */
-declare abstract class TypedArray<T> implements ArrayBufferView<T> {
+declare abstract class TypedArray<T> implements ArrayBufferView {
   [key: number]: T;
   /** Number of bytes per element. */
   static readonly BYTES_PER_ELEMENT: usize;
   /** Wrap an ArrayBuffer */
-  static wrap(buffer: ArrayBuffer, byteOffset?: i32, length?: i32): TypedArray<T>;
+  static wrap<T>(buffer: ArrayBuffer, byteOffset?: i32, length?: i32): TypedArray<T>;
```

Comment:
I see why this change was made, but it doesn't match the actual signature, which is non-generic on the concrete implementations, now.

Reply:
what if I change it to:
```
static wrap(buffer: ArrayBuffer, byteOffset?: i32, length?: i32): ArrayBufferView;
```
?